code/script.js

```diff
@@ -1,51 +1,152 @@
+const showMessage = (msg, sender) => {
+
+if (sender === 'user') {
+    chat.innerHTML += `<section class="user-msg">
+    <div class="bubble user-bubble">
+        <p>${msg}</p>
+    </div>
+    <img src="assets/user.png" alt="chat bot" />
+</section>`;
+} 
+else if (sender === 'bot') {
+    chat.innerHTML += `<section class="bot-msg">
+    <img src="assets/bot.png" alt="user bot" />
+    <div class="bubble bot-bubble">
+        <p>${msg}</p>
+    </div>
+</section>`;
+}
+
+  
+   //Makes sound when sending a message 
+  messageSound.currentTime = 0
+  messageSound.play()
+  
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight;
+};
+    
+//Eventlistener: Starts the function when user clicks submit. 
+form.addEventListener('submit', (event) => {
+  event.preventDefault()   //Stops it from autosaving.
+  
+  const firstName = document.getElementById('firstName').value;
+  console.log(firstName)           
+  
+// Calling the function "showMessage".
+  showMessage(firstName, 'user');    
+
+   //This code slows it with 1 sec.
+  setTimeout(() => recognize(firstName), 1000)  
+  
+})
+
+
+//Question 2
+
+const recognize = (firstName) => {
+  console.log(firstName);
+
+  chat.innerHTML += `<section class="bot-msg">
+    <img src="./bot.png" alt="user bot" />
+    <div class="bubble bot-bubble">
+        <p>How can I help you ${firstName}?</p>
+    </div>
+</section>`;
+
+
+  showMessage(`What a beautiful name, ${firstName}! Which book would you like?`, 'bot');
+
+        console.log(formWrapper);
+    formWrapper.innerHTML = 
+    `<button id="yogaBtn">How to Yoga book</button>
+    <button id="cookBtn">How to cook book</button>
+    <button id="englishBtn">Learn English book</button>`;
+  
+  
+    let yogaBtn = document.getElementById('yogaBtn');
+    yogaBtn.addEventListener('click', () => choices())
+  
+    let cookBtn = document.getElementById('cookBtn');
+    cookBtn.addEventListener('click', () => choices())  
+  
+    let englishBtn = document.getElementById('englishBtn');
+    englishBtn.addEventListener('click', () => choices())
+
+  
+}
+
+
+// Question number 3
+const choices = (msg) => {
+
+  showMessage(`Wow, love that your into learning stuff! What else?`, 'bot');
+  
+  formWrapper.innerHTML =
+        `<button id="boundBtn">Bound Book</button>
+      <button id="pocketBtn">Pocket Book</button>
+  <button id="neverMindBtn">Changed my mind</button>`
+  
+  let boundBtn = document.getElementById('boundBtn');
+  boundBtn.addEventListener('click', () => thankYou())
+
+  let pocketBtn = document.getElementById('pocketBtn');
+  pocketBtn.addEventListener('click', () => thankYou())  
+
+  let neverMindBtn = document.getElementById('neverMindBtn');
+  neverMindBtn.addEventListener('click', () => changedMind())
+
+
```

Comment:
Good and clear naming in this function. Maybe you should try and indent this code a bit better?